File: src/Messenger/Server/Messenger.Application/ViewModels/UserProfileViewModel.cs
namespace Messenger.Application.ViewModels
{
    public class UserProfileViewModel
    {
        public long Id { get; set; }
        public string FirstName { get; set; }
        public string PhoneNumber { get; set; }
        public string? LastName { get; set; }
        public string? Username { get; set; }
        public string? Email { get; set; }
        public string? PhotoPath { get; set; }
        public List<ChatViewModel> AuthorshipChats { get; set; }
        public DateTime? CreatedAt { get; set; }
    }
}

File: src/Messenger/Client/Messenger.UI/src/app/Interfaces/Users/user-profile-view-model.ts
import { Chat } from "../chat";

export interface UserProfileViewModel {
    id:number,
    firstName:string,
    phoneNumber:string,
    lastName:string|null,
    userName:string|null,
    email:string|null,
    photoPath:string|null,
    authorshipChats:Chat[],
    createdAt:Date
}
